core/drivers/stm32_bsec.c

```diff
@@ -160,6 +160,30 @@ static uint32_t bsec_status(void)
 	return io_read32(bsec_base() + BSEC_OTP_STATUS_OFF);
 }
 
+static bool state_is_invalid_mode(void)
+{
+	return bsec_status() & BSEC_MODE_INVALID;
+}
+
+static bool state_is_secured_mode(void)
+{
+	return bsec_status() & BSEC_MODE_SECURED;
+}
+
+static bool state_is_closed_mode(void)
+{
+	uint32_t close_mode = 0;
+	const uint32_t mask = CFG0_CLOSED_MASK;
+
+	if (IS_ENABLED(CFG_STM32MP13))
+		return bsec_status() & BSEC_MODE_CLOSED;
```

Comment:
I suggest you move these 2 lines into commit "drivers: stm32_bsec: update for stm32mp13".

Reply:
Ok

Reply:
Moving also 
`#define BSEC_MODE_CLOSED		BIT(8)`